### ISSUE TITLE
About LLaMA-3-LLaVA-NeXT-8B: The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

### ISSUE THREAD

**Issue by wangtong627** [LLaVA-VL/LLaVA-NeXT#194]
I am trying to use the llama3-llava-next-8b model, and I replaced --model-path with the local path of llama3-llava-next-8b that I downloaded. 
When I run python -m llava.serve.model_worker --host 0.0.0.0 --controller http://localhost:10000 --port 40000 --worker http://localhost:40000 --model-path '/to/my/path/llama3-llava-next-8b', I encountered the following error. Could you please help me resolve this issue?

The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.
![image](https://github.com/user-attachments/assets/2e0ba2d3-1815-49ac-8628-3c3dd4bbb409)

**Comment by Mike-ihr** [comment]
for this question, I recommend that you refer to this link https://github.com/microsoft/LLaVA-Med/issues/86
